inc/azure_c_shared_utility/constbuffer_array_batcher.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#ifndef ZRPC_BATCH_H
```

Comment:
>ZRPC [](start = 8, length = 4)

constbuffer_array_batcher

Reply:
Fixed, thanks!

---
In reply to: [324394196](https://github.com/Azure/azure-c-shared-utility/pull/375#discussion_r324394196) [](ancestors = 324394196)